packages/opc-footer/README.md

```diff
@@ -1,63 +1,62 @@
-# This is opc-footer 👋
+# opc-footer component 👋
 
-![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+![Version](https://img.shields.io/badge/version-0.0.2-blue.svg?cacheSeconds=2592000)
 ![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)
 [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/graphs/commit-activity)
 
-opc-footer is a web component based on lit-element. By default it has the light theme and text for copyright.
+A web component based on lit-element which provides themes and an api to show links. 
```

Comment:
A standardized web component based on Lit Element for Red Hat One Platform Footer.